plugins/repository-s3/src/main/java/org/elasticsearch/cloud/aws/InternalAwsS3Service.java

```diff
@@ -101,6 +101,7 @@ private synchronized AmazonS3 getClient(String endpoint, Protocol protocol, Stri
         if (maxRetries != null) {
             // If not explicitly set, default to 3 with exponential backoff policy
             clientConfiguration.setMaxErrorRetry(maxRetries);
+            clientConfiguration.setUseThrottleRetries(AwsS3Service.CLOUD_S3.THROTTLE_RETRIES_SETTING.get(settings));
```

Comment:
Shouldn't this be set independently from the `maxRetries` setting? I think the default retry policy is 3 and users might want to throttling this too event if the `max_retries` is not set in the repository settings.